tests/unit/NetworkTest.ts

```diff
@@ -50,7 +53,7 @@ afterEach(() => {
 });
 
 describe('NetworkTests', () => {
-    test('failing to reauthenticate while offline should not log out user', () => {
```

Comment:
Removed this part of the description because this test doesn't turn off the network